### PR TITLE
fix: "+" button for unit 

### PR DESCRIPTION
**Pull request by sophieyang0813**
This adds "+" button docked to the bottom of the unit screen. It partly addresses #370. Other #370 related changes will be followed in consequent pull requests.

### CODE REVIEW

File: imports/ui/unit-explorer/unit-explorer.jsx [hunk 1]
```diff
@@ -43,13 +50,17 @@ class UnitExplorer extends Component {
...
               </div>
             </div>
-            <Link to='/unit/new' className='link'>
-              <MenuItem className='no-shrink' innerDivStyle={{padding: 0}}>
-                <div className='tc bondi-blue fw5 bt b--moon-gray'>
-                  Add Unit
-                </div>
-              </MenuItem>
-            </Link>
+            {fabDescriptors.map((desc, ind) => (
```
Comment (nbiton) [inline imports/ui/unit-explorer/unit-explorer.jsx]: Using a map to iterate over an array of 1 is a bit confusing in this case. I know the original usage pattern you found this in includes it, but it doesn't make sense in this scenario, as there shouldn't be more than 1 FAB in any scenario.
You can skip the ".map" and just render the FAB directly... copy over the value from `fabDescriptors` to be used directly here

File: imports/ui/unit-explorer/unit-explorer.jsx [hunk 2]
```diff
@@ -43,13 +50,17 @@ class UnitExplorer extends Component {
...
+              <div key={ind} className='absolute bottom-2 right-2'>
+                <FloatingActionButton
+                  backgroundColor={desc.color}
+                  className='zoom-effect'
```
Comment (nbiton) [inline imports/ui/unit-explorer/unit-explorer.jsx]: You don't need 'zoom-effect' in this case. It was used to show the transition animation in the other place you found this,

**Comment by franck-boullier** [comment]
Button is not docked at the bottom right of the screen ---> button is moving up as we scroll down

image

**Comment by franck-boullier** [comment]
> Button is not docked at the bottom right of the screen ---> button is moving up as we scroll down

- Tested on Firefox on a PC: button is not docked (DT and mobile version)
- Tested on Chrome on Samsung: button is not docked

**Comment by sophieyang0813** [comment]
@franck-boullier @nbiton the button is docked on my browser. How can I debug in this case? 
screen shot 2018-07-10 at 9 33 59 am

**Comment by franck-boullier** [comment]
> How can I debug in this case?

Sorry I'm useless here 😊

File: imports/ui/unit-explorer/unit-explorer.jsx [hunk 3]
```diff
@@ -43,13 +49,14 @@ class UnitExplorer extends Component {
...
             </div>
-            <Link to='/unit/new' className='link'>
-              <MenuItem className='no-shrink' innerDivStyle={{padding: 0}}>
-                <div className='tc bondi-blue fw5 bt b--moon-gray'>
-                  Add Unit
-                </div>
-              </MenuItem>
-            </Link>
+            <div className='absolute bottom-2 right-2'>
+              <FloatingActionButton
```
Comment (nbiton) [inline imports/ui/unit-explorer/unit-explorer.jsx]: You should use the values from `fabDescriptor` directly here, and remove it.
The added benefit of that is that when you look at this code later, you won't need to scroll up and look for `fabDescriptor`. You'll know what this button does just by looking at these lines.

**Comment by nbiton** [comment]
@sophieyang0813 I think you should add enough units until the list becomes long enough and you need to scroll

File: imports/ui/case-explorer/case-explorer.jsx [hunk 4]
```diff
@@ -71,6 +71,7 @@ class CaseExplorer extends Component {
       })
     }
   }
+
```
Comment (nbiton) [inline imports/ui/case-explorer/case-explorer.jsx]: The empty lines should not be committed (there are no other changes in this file)

**Comment by sophieyang0813** [comment]
@nbiton This screenshot captures all the units under Leonel and the button is docked in my browser. I wish my screenshot can capture the scroll bar. 
screen shot 2018-07-11 at 7 17 19 pm
screen shot 2018-07-11 at 7 17 12 pm

**Comment by nbiton** [comment]
@sophieyang0813 yeah, it looks to work as expected for me too, actually... @franck-boullier ?

**Comment by franck-boullier** [comment]
Let me have a quick look

**Comment by franck-boullier** [comment]
This seems to work on:
- Chrome on my DT 
- Chrome as a simulated mobile phone on my DT
- Chrome on my Galaxy S8
- Samsung default browser on my Galaxy S8

I'm OK with that so you can merge once all the conflicts and requested changes are done even if this is still f... up in FF on a simulated mobile for some weird reason...
image

**Comment by sophieyang0813** [comment]
@franck-boullier Let me have a look again

**Comment by franck-boullier** [comment]
@Sophie at this point I'd prefer that younspend time on more important issues.
This is good enough ---> let's merge this and move on to 374 and also issue 197

**Comment by nbiton** [comment]
Once the issues are addressed and the conflicts are resolved, we can merge this

**Comment by franck-boullier** [comment]
@nbiton OK to merge?

**Comment by nbiton** [comment]
No...
* **case-explorer.jsx** is still included in the changed files list, even though it only has one added empty line (the file shouldn't appear at all)
* My comment regarding **unit-explorer.jsx** has not been addressed yet

**Comment by franck-boullier** [comment]
@sophieyang0813
- Which commit is addressing this "comment regarding unit-explorer.jsx " from @naor
- Which commit is addressing this "case-explorer.jsx is still included in the changed files list" from @naor
Thanks

**Comment by franck-boullier** [comment]
@sophieyang0813 when you have finished all your changes, please add a comment so we know that this should be reviewed (and ideally what should we review too)

**Comment by nbiton** [comment]
The "className" attribute wasn't affecting the button in this case, but it's still in the desired color anyway. I removed "className" from the button.

**Pull request merged**